### PR TITLE
[bitbucket-server] reuse icon from bitbucket

### PR DESCRIPTION
**Pull request by AlexTugarev**
A quick fix for the login page.

fixes https://github.com/gitpod-io/gitpod/issues/8603

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@AlexTugarev: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by gtsiolis** [comment]
Looking at this now! 👀

File: components/dashboard/src/provider-utils.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ function iconForAuthProvider(type: string) {
             return <img className="fill-current filter-grayscale w-5 h-5 ml-3 mr-3 my-auto" src={gitlab} />;
         case "Bitbucket":
             return <img className="fill-current filter-grayscale w-5 h-5 ml-3 mr-3 my-auto" src={bitbucket} />;
+        case "BitbucketServer":
+            return <img className="fill-current filter-grayscale w-5 h-5 ml-3 mr-3 my-auto" src={bitbucket} />;
```
Comment (gtsiolis) [inline components/dashboard/src/provider-utils.tsx:21]: LGTM! See also [relevant discussion](https://gitpod.slack.com/archives/C02EN94AEPL/p1649068086035109?thread_ts=1649063420.987519&cid=C02EN94AEPL) (internal).

**Review by gtsiolis — APPROVED** [review]

**Review by jldec — APPROVED** [review]
LGTM (could not test because preview env not configured for BBS)

**Pull request merged**